js/state/siteUtil.js

```diff
@@ -607,7 +607,7 @@ module.exports.isHistoryEntry = function (siteDetail) {
       // This is a Brave default newtab site
       return false
     }
-    return !!siteDetail.get('lastAccessedTime') && !isBookmarkFolder(siteDetail.get('tags'))
+    return !isBookmarkFolder(siteDetail.get('tags'))
```

Comment:
I think this change causes all bookmarks to show up as History items.
It might be that https://github.com/brave/browser-laptop/blob/master/app/common/lib/historyUtil.js#L19 is now returning every site not a bookmark folder.

Repro:
1. Start pyramid 0, enable Sync and visit archive.org and wikipedia.org
2. Start pyramid 1, join pyramid 0's sync group
3. Examine pyramid 1's history – it has the 2 sites.

Expected: Pyramid 1's history should be empty because history doesn't sync by default.